artifacts/definitions/Windows/Events/Windows.Alerts.ProcessCreation.yaml

```diff
@@ -0,0 +1,56 @@
+name: Windows.Alerts.ProcessCreation
+description: |
+  This artifact logs specific process creation events to Velociraptor. It watches the Sysmon ETW provider for new events. It requires Sysmon to be installed on endpoints, you can install it by running a hunt with the artifact 'Windows.Sysinternals.SysmonInstall' (the default Sysmon config will suffice, otherwise make sure it at least logs Sysmon Event ID 1).
```

Comment:
I dont think this is necessary actually because SysmonLogForward already ensures sysmon is installed